wire-ios/Wire-iOS/Sources/Authentication/Backup/BackupRestoreController.swift

```diff
@@ -40,7 +40,11 @@ final class BackupRestoreController: NSObject {
     // their attachments and change the underscore with a dash. This is the reason we accept 2 types
     // of file extensions: 'ios_wbu' and 'ios-wbu'.
 
-    static let WireBackupUTIs = ["com.wire.backup-ios-underscore", "com.wire.backup-ios-hyphen"]
+    static let WireBackupUTIs = [
+        "com.wire.backup-universal",
```

Comment:
I just chose any string for this identifier. Shall or do we need to align with other platforms?

Reply:
This value is only relevant internally to the iOS app right? If so, this is fine.

Reply:
I think so, yes.